### PR TITLE
Create App structure and four components

### PR DESCRIPTION
**Pull request by IBTechRaj**
- Milestone-2 tasks completed
- Created components folder
- Created four files in components folder and created basic Calculator project structure

Submitting for code review. 
Thanks

### CODE REVIEW

File: src/components/Display.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from "react"
+
+function Display() {
+  return (
+    <input type='text'/>
+    
+  )
+}
+
```
Comment (shloch) [inline src/components/Display.js]: - this component should have an incoming prop called `result` [see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/incomingProps.png)]
- `result` should be a string (validate this with `propTypes`) [see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/validateAsString.png)]

- `result` should have default value of `"0"` (enforce this with `defaultProps`) [see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/DefaultProp.png)]

File: src/components/Button.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React from "react"
+
+const  Button = props => {
+  return (
+        <div>{props.children}</div>
+  )
+}
```
Comment (shloch) [inline src/components/Button.js]: - this component should accept an incoming `name` prop [see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/button/acceptProp.png)]

- `name` prop should be string (enforce this with `propTypes`) [see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/button/validateAsString.png)]

File: src/components/ButtonPanel.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+            <Button >2</Button>
+            <Button >3</Button>
+            <Button >+</Button>
+          </div>
+          <div className="row">
+            <Button >0</Button>
+            <Button >.</Button>
+            <Button >=</Button>
+          </div>
+          </div>
```
Comment (shloch) [inline src/components/ButtonPanel.js]: From my comment above (regarding the `<Button />` component,

this implies, you should render `<Button />` component here and pass a `name` prop

File: src/components/Button.js (hunk 2 above)
Comment (shloch) [reply src/components/Button.js]: It's advisable to not use the `React children` approach here........

**Review by shloch — CHANGES_REQUESTED** [review]
## Hello @IBTechRaj , 

We're advancing well, good job so far with what you've brought up
But we have some debugging to do......... (see comments below)

### Happy coding

**Comment by shloch** [comment]
Also, fix any `Stickler` error that you have

**Comment by IBTechRaj** [comment]
Made all the corrections suggested.

Submitting for rereview of code

**Review by bellom — APPROVED** [review]
## Hi @IBTechRaj  :clap: 

**Great work making all the necessary corrections from the previous review.**

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit project completion form.
Good luck in the next milestone :slightly_smiling_face:_ 

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**